pkg/utils/host/workaround_nonlinux.go

```diff
@@ -0,0 +1,33 @@
+//go:build !linux
+// +build !linux
```

Comment:
The build flags here are a workaround for https://github.com/inspektor-gadget/inspektor-gadget/issues/1830